src/components/JobSalaryDetails.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { View, Text } from "native-base";
+import FontAwesome from "react-native-vector-icons/FontAwesome";
+import styles from "../styles";
+import { COLOR } from "../styles/color";
+
+const JobSalaryDetails = props => {
+  return (
+    <View style={styles.viewDesign}>
+      {props.children}
+      <View style={styles.iconView}>
+        <View style={styles.viewRow}>
+          <FontAwesome
+            name="briefcase"
+            color={COLOR.Red}
+            style={styles.viewIcon}
+          />
+          <Text style={styles.rowText}>0 - 1 years</Text>
```

Comment:
don't hard code it here. pass via props

Reply:
sir, this component is just a single line, used in other component with the same data that's why it is hardcoded.